ansible/hosts/lxd_hosts.txt

```diff
@@ -2,25 +2,26 @@
 [cs-servers]
 xroad-lxd-cs ansible_connection=lxd
 
-#configuration proxies (ubuntu lxd containers)
+#configuration proxies (ubuntu lxd containers, optional)
 [cp-servers]
-xroad-lxd-cp ansible_connection=lxd
+#xroad-lxd-cp ansible_connection=lxd
 
 #certification authority, time stamping authority and ocsp service server for testing purposes (ubuntu)
 [ca-servers]
-xroad-lxd-ca ansible_connection=lxd
+xroad-lxd-cs ansible_connection=lxd
```

Comment:
Should it be `xroad-lxd-ca`?

Reply:
The ca role can be installed on the cs instance, avoiding one container --> saves some resources if the development machine is low on resources.